spring/src/test/java/org/axonframework/spring/eventsourcing/SpringAggregateSnapshotterFactoryBeanTest.java

```diff
@@ -76,7 +76,7 @@ public StubAggregate doCreateAggregate(String aggregateIdentifier,
         mockTransactionManager = mock(PlatformTransactionManager.class);
         aggregateIdentifier = UUID.randomUUID().toString();
 
-        String type = "testAggregate";
+        String type = "StubAggregate";
```

Comment:
Would be nice for the test to pass without modifications, to show backwards compatibility.